### PR TITLE
Add colors on the get_operations command

### PR DESCRIPTION
**Pull request by litchipi**
Implements #3294 for the REPL `get_operations`

Adds color on the CLI output to display more effectively operations informations.

Screenshot  Capture d’écran du 2023-03-08 11-46-08

Try it on your own and tell me if you want some other colors.

### CODE REVIEW

**Comment by modship** [comment]
Hi @litchipi, nice ! 

I think it's better to have one PR with all client cmd we should color. And create 1 PR for each cmd to merge in the first branch. 

in this way, all color cli cmd will be merge in same time and not step by step.

What do you think ?

File: massa-client/src/repl.rs [hunk 1]
```diff
@@ -351,8 +351,41 @@ impl Output for Vec<IpAddr> {
 
 impl Output for Vec<OperationInfo> {
     fn pretty_print(&self) {
-        for operation_info in self {
-            println!("{}", operation_info);
+        for info in self {
+            print!("Operation {}", style(info.id).color256(175));
```
Comment (modship) [inline massa-client/src/repl.rs:355]: it is possible to have the color in enum or constant ?

**Comment by litchipi** [comment]
Seams like a nice idea ! I'm gonna do this now

File: massa-client/src/repl.rs (hunk 1 above)
Comment (litchipi) [reply massa-client/src/repl.rs:355]: For sure, I'll try to do something more generic to have something more elegant

**Comment by litchipi** [comment]
I did something wrong it closed the PR
Sorry I'm gonna re-open a new one :-|

**Pull request closed without merging**